app/models/wpcc/legal_period_percents.rb

```diff
@@ -0,0 +1,61 @@
+module Wpcc
+  class LegalPeriodPercents
+    include ActiveModel::Model
+    attr_accessor :name,
+                  :employee_percent,
+                  :employer_percent,
+                  :tax_relief_percent
+
+    PERIODS_FILE = Wpcc::Engine.root.join(
+      'config',
+      'legal_periods_percents.yml'
+    )
+    PERIODS = HashWithIndifferentAccess.new(YAML.load_file(PERIODS_FILE))
+
+    def self.all
+      PERIODS.map do |period, percents|
+        new(
+          name: period.to_s,
+          employee_percent: percents[:employee_percent],
+          employer_percent: percents[:employer_percent],
+          tax_relief_percent: percents[:tax_relief_percent]
+        )
+      end
+    end
+
+    def self.filter(employee_percent, employer_percent)
```

Comment:
Yes user input employee percent is more clear. I was struggle with this name too. Because is a user contribution percent and the user employer contribution percent. I will renamed it.